### DISCUSSION TITLE
Vibe Check

### DISCUSSION THREAD

**Discussion by lyxal** [Vyxal/Vyxal#468] [Q&A]
So recently there was a discussion in the Nineteenth Byte about the fact there's cognitive dissonance (conflicting portrayals of the attitude/image of Vyxal) within our operations. I want to know what y'all think about that. More specifically:

1. How do you currently perceive the "atmosphere" of Vyxal? E.g. do you believe there's too much of a blur between the serious goals of Vyxal and some of the nonsense that happens?
2. What direction would _you_ like to see Vyxal take? E.g. more seriousness? Less seriousness? More organisation? Less?
3. What do you think of the level of advertising of Vyxal? E.g. is too much advertising happening?
4. Since the last time we discussed the future of Vyxal, do you feel that what you suggested has been implemented? [The discussion](https://github.com/Vyxal/Vyxal/discussions/191)
5. Do you have any other comments/anything else you want to mention?

Please answer as honestly if you can - if there's things about the way we currently do things you don't like or things you want improved, I want to know. That is, don't just write things you think I want to hear, and don't shy away from leaving constructive criticism.

**Comment by cgccuser** [comment]
I like the atmosphere of Vyxal personally. I like the occasional sh\*tposting because it lightens the mood, and I like the organization we now have because it makes it easier to get things done. Vyxal still has the "small, fun project" feel to me, the PRs and reviews and everything haven't changed that.

I do feel that Lyxal specifically treats Vyxal more seriously than the rest of us (despite being the primariy sh\*tposter :P), but that's understandable, because it's their language after all. We could cut down on the advertising a teeny bit (maybe only suggest Vyxal if someone asks for a good golflang they should learn, and don't be *too* insistent about it). Also, I think the documents/philosophy folder is a bit much. I can't tell if it's half ironic or what, but quite a bit of it is in the README, and When.md, Where.md, and Why.md feel unnecessary to me.

tbh though, I think Vyxal's doing fine, don't heed Redwolf too much. Of course, if Redwolf feels Vyxal is too "corporate," then other newcomers might too, and we should do a little work on making it seem more like the friendly little community it is, but it's not a huge problem imo.

**Reply by lyxal** [reply]
The philosophy folder was meant to be a sort of inspirational thing during the rewrite. But seeing as how that's finished, I'd say that it would be reasonable to delete the files now.

**Comment by pxeger** [comment]
> the fact there's cognitive dissonance

false

**Reply by cgccuser** [reply]
But perhaps you have cognitive dissonance about there not being cognitive dissonance :P

**Reply by pxeger** [reply]
almost like that was the joke

**Reply by cgccuser** [reply]
I don't believe you

**Comment by mathcat4** [comment]
So, first of all, I want to make it clear that I am not that person, who actively contributed to Vyxal or is actively using Vyxal.
Heck, I wasn't even there when these discussions happened. These are only my opinions:

So about the nonsense and the atmosphere:
I think the atmosphere's great. When I wanted to learn Vyxal all of the Vyxalers welcomed me kindly to learn Vyxal. I feel it's still like this now. About this nonsense and stuff, I think lyxal is talking about the SE chat. But I'm with Fred and George:
> Nonsense is the pinch of salt that makes life delicious.

(No, I just made it up)

I like Vyxal and the Vyxalers as it is: Nonsensive, but still serious. Relaxed, but with the goal in mind.
Advertising ... I don't know. I know that advertising may introduce people to golflangs and other stuff, I can understand advertising in SE and ... I don't know.
As cgccuser, I think Vyxal's fine too and I really can't tell why Redwolf feels Vyxal's too corporate.
